src/main/java/me/zeroeightsix/fiber/tree/Property.java

```diff
@@ -15,7 +15,16 @@
      *
      * @param value the new value this property should receive
      * @return {@code true} if this property changed as a result of the call
+     * @see #accepts(Object)
      */
     boolean setValue(T value);
 
+    /**
+     * Returns {@code true} if the given value satisfies this property's constraints.
+     *
+     * @param value the value to check
+     * @return {@code true} if the given value satisfies this property's constraints, {@code false} otherwise.
+     * @see #setValue(Object)
+     */
+    boolean accepts(T value);
```

Comment:
Can you default this method to return `true`? Implementing `Property` doesn't mean a class is capable of constraints.